### PR TITLE
Added base url to the ui tests

### PR DESCRIPTION
**Pull request by ajitava-git**
Have added the base URL to the UI tests. The base URL is not added in the API test.
Have executed the test_example_form.py and verified.

### CODE REVIEW

File: tests/test_mobile_bitcoin_price.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def test_mobile_bitcoin_price(test_mobile_obj):
 
     # Run  the test only if the options provided are valid.
     if options_obj.check_options(options):
-        test_mobile_obj = PageFactory.get_page_object("Zero mobile")
+        test_mobile_obj = PageFactory.get_page_object("Zero mobile", base_url=test_obj.base_url)
```
Comment (qxf2) [inline tests/test_mobile_bitcoin_price.py:79]: This line is throwing an error: 

tests/test_mobile_bitcoin_price.py::test_mobile_bitcoin_price Exception when trying to run test:/root/project/tests/test_mobile_bitcoin_price.py
Python says:local variable 'test_obj' referenced before assignment

For now, can we just revert the changes within this mobile test alone?

**Review by qxf2 — CHANGES_REQUESTED** [review]
@ajitava-git - the CI test for mobile automation is failing. Can you please check if it is because of your code. If yes, please revert the change in that file alone.

This is the error I see on CircleCI:

"tests/test_mobile_bitcoin_price.py::test_mobile_bitcoin_price Exception when trying to run test:/root/project/tests/test_mobile_bitcoin_price.py
Python says:local variable 'test_obj' referenced before assignment"

File: tests/test_mobile_bitcoin_price.py (hunk 1 above)
Comment (ajitava-git) [reply tests/test_mobile_bitcoin_price.py]: @qxf2 - I have reverted the change and pushed it.
